unit-10/script.js

```diff
@@ -0,0 +1,188 @@
+// lesson 1:
+// In một bảng nhân của một số bất kỳ với các số từ 1 tới 10 và hiển thị kết quả
+
+function getMultiplyTable(number) {
```

Comment:
```suggestion
function getMultiplyTable (number) {
```
Khi khai báo hàm thì thêm 1 dấu cách trước `(`